src/components/Spawn/editHostModal/getFormSchema.tsx

```diff
@@ -0,0 +1,297 @@
+import { css } from "@emotion/react";
+import { add } from "date-fns";
+import widgets from "components/SpruceForm/Widgets";
+import { AntdSelect } from "components/SpruceForm/Widgets/AntdWidgets";
+import { LeafyGreenTextArea } from "components/SpruceForm/Widgets/LeafyGreenWidgets";
+import { InputLabel, StyledLink } from "components/styles";
+import { windowsPasswordRulesURL } from "constants/externalResources";
+import { GetMyPublicKeysQuery, MyVolumesQuery } from "gql/generated/types";
+import { GetFormSchema } from "pages/projectSettings/tabs/types";
```

Comment:
Same comment about this we shouldn't be importing this from the `pages/projectSettings` folder.